client/src/rpc_client.rs

```diff
@@ -106,6 +111,21 @@ impl RpcClient {
         )
     }
 
+    pub fn new_with_timeouts_and_commitment(
```

Comment:
Could make the old method call this one to dereplicode a bit

Reply:
I decided to stick with `new_sender` in the old method, because I didn't want this method to have to take an Option<Duration>. Seems okay for now.